src/wolfcrypt/ciphers.py

```diff
@@ -435,11 +435,18 @@ def __init__(self, key = None):  # pylint: disable=super-init-not-called
 
             if key != None:
                 key = t2b(key)
-
                 ret = _lib.wc_RsaPrivateKeyDecode(key, idx,
                                               self.native_object, len(key))
                 if ret < 0:
-                    raise WolfCryptError("Invalid key error (%d)" % ret)
+                    idx[0] = 0
```

Comment:
Per some feedback from the ZD ticket, I opted to not add a pkcs8 parameter and instead just try to decode the key in pkcs8 format if the traditional decoding fails. Makes the interface a bit cleaner.

Reply:
Sounds good, thanks